functions/Connect-DbaInstance.ps1

```diff
@@ -147,6 +147,12 @@ function Connect-DbaInstance {
         Connect to an Azure SQL Database or an Azure SQL Managed Instance with an AccessToken, that has to be generated with Get-AzAccessToken.
         Note that the token is valid for only one hour and cannot be renewed automatically.
 
+    .PARAMETER DedicatedAdminConnection
+        Connects using "ADMIN:" to create a dedicated admin connection (DAC).
+        If the instance is on a remote server, the remote access has to be enabled via "sp_configure 'remote admin connections', 1".
+        The parameter NonPooledConnection will be set to request a non-pooled connection.
+        The connection will not be closed if the variable holding the Server SMO is going ot of scope, so it is very important to call .ConnectionContext.Disconnect() to close the connection.
```

Comment:
*out* of scope

Reply:
Also, can we do this/detect idle in runspaces? Can we create a `Disconnect-DbaInstance` command?

Reply:
`Disconnect-DbaInstance` is a good idea. But for that we would have to build a cache (like for TEPP) where Connect-DbaInstance saves (registers) all connections it opens. Then `Disconnect-DbaInstance -SqlInstance XYZ` could get all connections from cache and close them. `Disconnect-DbaInstance -SqlInstance XYZ -Force` could connect to the instance and get all connections based on the client name and kill them.